vcd/resource_vcd_vapp_vm.go

```diff
@@ -967,13 +989,13 @@ func genericResourceVmCreate(d *schema.ResourceData, meta interface{}, vmType ty
 	return nil
 }
 
-// createVmFromTemplate is responsible for create VMs from template of two types:
+// createVmFromImage is responsible for create VMs from template of two types:
 // * Standalone VMs
 // * VMs inside vApp (vApp VMs)
 //
 // Code flow has 3 layers:
 // 1. Lookup common information, required for both types of VMs (Standalone and vApp child). Things such as
-//   - Template to be used
+//   - VM Source Image - Catalog Template or VM (VM Copy operation) to be used
```

Comment:
Maybe this for more clarity:

```suggestion
//   - VM Source Image - Catalog Template or VM (for VM Copy operation) to be used
```

Reply:
Fixed.